requirements/pip.txt

```diff
@@ -72,19 +72,19 @@ defusedxml==0.6.0
 
 # Utils
 django-gravatar2==1.4.2
-pytz==2019.2
+pytz==2019.3
 Unipath==1.1
 django-kombu==0.9.4
 mock==3.0.5
-stripe==2.35.1
+stripe==2.37.2
 
 # unicode-slugify==0.1.5 is not released on PyPI yet
 git+https://github.com/mozilla/unicode-slugify@b696c37#egg=unicode-slugify==0.1.5
 
 django-formtools==2.1
 django-crispy-forms==1.7.2
 
-docker==4.0.2
+docker==4.1.0
```

Comment:
I don't think this will be a problem, but we should be careful when deploying it.